src/js/Subway.js

```diff
@@ -0,0 +1,5 @@
+export class Subway {
```

Comment:
How are you going to modify this? You don't pass you lines into constructor and do not have methods for modification...